joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/AstNodeBuilder.scala

```diff
@@ -46,7 +47,7 @@ trait AstNodeBuilder[Node, NodeProcessor] { this: NodeProcessor =>
 
   protected def unknownNode(node: Node, code: String): NewUnknown = {
     NewUnknown()
-      .parserTypeName(node.getClass.getSimpleName)
+      .parserTypeName(Try(node.getClass.getSimpleName).toOption.getOrElse(Defines.Unknown))
```

Comment:
I see `node` is also passed to `line` and `column` etc. If `node == null` then this may be problematic (similarly for other calls too). Might be worth doing this kind of `Try` check for `line` calls too.

No need to handle nulls for the other calls for now I'd say

Reply:
for `line` handling is already present

```
  override def line(element: PsiElement): Option[Integer] = {
    try {
      Some(
        element.getContainingFile.getViewProvider.getDocument
          .getLineNumber(element.getTextOffset) + 1
      )
    } catch {
      case _: Throwable => None
    }
  }
  ```

Reply:
similarly present for `column` as well